Dockerfile

```diff
@@ -76,13 +74,21 @@ RUN source $HOME/.cargo/env && \
     cp target/$RUST_TARGET/release/twilight-http-proxy /twilight-http-proxy && \
     actual-strip /twilight-http-proxy
 
-FROM docker.io/${FINAL_TARGET}/alpine:latest
-ARG TARGET
+FROM docker.io/library/alpine:edge AS dumb-init
```

Comment:
I'd like to pick a different name for this concept than "dumb"

Reply:
The program is actually called like that and is a fully static init system. See https://github.com/Yelp/dumb-init

Reply:
Oh. lol